src/main/java/io/micronaut/build/MicronautGradleEnterprisePlugin.java

```diff
@@ -82,20 +80,14 @@ private void configureGradleEnterprise(Settings settings,
                 }
                 buildCache.getLocal().setEnabled(localEnabled);
                 if (remoteEnabled) {
-                    buildCache.remote(HttpBuildCache.class, remote -> {
-                        remote.setUrl("https://ge.micronaut.io/cache/");
+                    buildCache.remote(ge.getBuildCache(), remote -> {
                         remote.setEnabled(true);
                         if (push) {
-                            String username = envOrSystemProperty(providers, "GRADLE_ENTERPRISE_CACHE_USERNAME", "gradleEnterpriseCacheUsername", "");
-                            String password = envOrSystemProperty(providers, "GRADLE_ENTERPRISE_CACHE_PASSWORD", "gradleEnterpriseCachePassword", "");
-                            if (!username.isEmpty() && !password.isEmpty()) {
+                            String accessKey = providers.systemProperty("GRADLE_ENTERPRISE_ACCESS_KEY").getOrNull();
+                            if (accessKey != null && !accessKey.isEmpty()) {
                                 remote.setPush(true);
-                                remote.credentials(creds -> {
-                                    creds.setUsername(username);
-                                    creds.setPassword(password);
-                                });
                             } else {
-                                System.err.println("WARNING: No credentials for remote build cache, cannot configure push!");
+                                System.err.println("WARNING: Access key missing for remote build cache, cannot configure push!");
```

Comment:
I think the string can be left the same. They are both still credentials at the end of the day.

Reply:
That is correct, however changing it is clearer for the user as to what's missing.